CustomerApp/CustomerApp/Pages/YourOrderPage.xaml

```diff
@@ -10,45 +10,103 @@
             x:Name="yourOrderLayout">
             <!--Stack layout to place items into-->
-            <ListView
-                  x:Name="menuFoodItemsView"
-                  VerticalOptions="FillAndExpand"
-                  HorizontalOptions="CenterAndExpand"
-                  Margin="40,80,40,20"
-                  HasUnevenRows="True">
+            
+            <RefreshView
+                x:Name="orderRefreshView"
+                Command="{Binding onRefresh()}"
+                VerticalOptions="Fill"
+                HorizontalOptions="CenterAndExpand"
+                Margin="40,60,40,10">
+                <ListView
+                      x:Name="menuFoodItemsView"
+                    VerticalOptions="Fill"
+                      HasUnevenRows="True"
+                    SelectionMode="None">
+                    
                 <ListView.ItemTemplate>
                     <DataTemplate>
-                        <ViewCell>
-                            <Grid>
-                                <Grid.RowDefinitions>
-                                    <RowDefinition Height="Auto" />
-                                    <RowDefinition Height="Auto"/>
-                                </Grid.RowDefinitions>
-                                <StackLayout
-                                          x:Name="menuFoodItemLayout"
-                                          Grid.Row="0"
-                                          Orientation="Horizontal">
-                                    <Label
-                                                x:Name="uxMenuItemLabel"
-                                                FontSize="Medium"
-                                                Text="{Binding name}"/>
-                                    <Label
-                                                x:Name="uxMenuItemPriceLabel"
-                                                HorizontalOptions="EndAndExpand"
-                                                FontSize="Medium"
-                                                Text="{Binding StringPrice}"/>
-                                </StackLayout>
-                                <Label
-                                        x:Name="uxMenuItemSpcInst"
-                                        Grid.Row="1"
-                                        FontSize="Medium"
-                                        VerticalTextAlignment="Start"
-                                        Text="{Binding special_instruct}"/>
-                            </Grid>
-                        </ViewCell>
-                    </DataTemplate>
+                            <ViewCell>
+
+                                <Frame
+                                        CornerRadius="10"
+                                        Padding="0"
+                                        Margin="0,10,0,0"
+                                        BorderColor="Transparent">
+                                    <SwipeView
```

Comment:
I think something odd might be going on here. It almost seems like after one partial swipe, the swipe functionality gets disabled.